File: benchmarks/loops/bench.zig
const std = @import("std");
fn loop_sum() i64 {
    var sum: i64 = 0;
    var i: i64 = 0;
    while (i < 10000000) {
        sum += i;
        i += 1;
    }
    return sum;
}
pub fn main() void {
    std.debug.print("{d}\n", .{loop_sum()});
}

File: benchmarks/loops/bench.rs
fn loop_sum() -> i64 {
    let mut sum: i64 = 0;
    let mut i: i64 = 0;
    while i < 10000000 {
        sum += i;
        i += 1;
    }
    sum
}
fn main() {
    println!("{}", loop_sum());
}
